src/paho/mqtt/__init__.py

```diff
@@ -1,4 +1,10 @@
-__version__ = "1.2.2"
+__version__ = "1.3.0.dev0"
```

Comment:
The next version is not yet defined to be 1.3 or 2.0.
That why I would not yet update the version.